python/cudf_polars/cudf_polars/dsl/expressions/boolean.py

```diff
@@ -31,22 +30,49 @@
 __all__ = ["BooleanFunction"]
 
 
+class BooleanFunctionName(Enum):
```

Comment:
*suggestion*: use `IntEnum`.

Also, let's nest this class inside the `BooleanFunction` class below, and just call it `Name`.

Reply:
Done in https://github.com/rapidsai/cudf/pull/17418/commits/3f69d98ed7e525aef6ac1456a5586a1ea12f28ff .